src/focus_manager/__tests__/focus-navigation-test.js

```diff
@@ -40,9 +40,17 @@ describe("complex navigation tests", () => {
   function testInput(currFocus, action, newFocus) {
     focusManager.setFocus(currFocus);
     focusManager.onInputAction(action);
+    if (!newFocus) newFocus = currFocus; // unspecified means no change to focus
     expect(focusManager.currentFocus).toBe(newFocus);
   }
 
+  function testAllInputs(currFocus, leftFocus, rightFocus, upFocus, downFocus) {
```

Comment:
Shame I couldn't do this easily on Roku with how the unit test framework is integrated.

Reply:
Really? It is just a function with 5 args. You existing verification calls could be moved to one, or does it need the @Params visibility?

Reply:
The problem was related to the scope of the assertion -- Moving the assertions out of the main test context broke its usage.  I might be able to do something like passing the context or specific assertion reference, something I can look at again at some point.